tasks/ermakov_a_numb_viol_elem_vec/report.md

```diff
@@ -0,0 +1,319 @@
+Подсчёт числа нарушений монотонности в векторе

```

Comment:
Please, add at least some formatting, it is difficult to read unstructured report